bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -151,11 +151,11 @@ metadata:
     categories: OpenShift Optional
     certified: "false"
     containerImage: quay.io/konveyor/oadp-operator:latest
-    createdAt: "2020-09-08T12:21:00Z"
+    createdAt: "2023-07-05T10:14:04Z"
```

Comment:
what command updates the operator files?

Reply:
I have used `make bundle`

Reply:
for the team: when is this field updated?

Reply:
it was populated when repo first initialized. Only recent operator-sdk version update this field after every `make bundle`. We have yet to upgrade this repo to the newer version.